[
    {
        "user": "U2J80GQLW",
        "text": "<@U2J80GQLW|rajkishan> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1475307148.001884"
    },
    {
        "user": "U2J8FABS9",
        "text": "<@U2J8FABS9|kerstentw> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1475314633.001885"
    },
    {
        "user": "U2J8C2KN2",
        "text": "<@U2J8C2KN2|harish> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1475316674.001886"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "Anyone around that can give a quick hand re using git-review for pushing changes...",
        "ts": "1475317291.001887"
    },
    {
        "user": "U2J6ZKX19",
        "text": "<@U2J6ZKX19|wanghaibo> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1475318930.001888"
    },
    {
        "user": "U2J9QLUA3",
        "text": "<@U2J9QLUA3|romanix> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1475335998.001889"
    },
    {
        "type": "message",
        "user": "U294C6FDW",
        "text": "I made a quick change in the chaincode.go  to add         rowBytes, err := stub.GetState(keyString)\n        if err != nil {\n                return row, fmt.Errorf(\"Error fetching row from DB: %s\", err)\n        }\n        if rowBytes == nil {\n                return row, fmt.Errorf(\"Error no  row found from DB: %s\", err)\n        }",
        "attachments": [
            {
                "fallback": "[September 30th, 2016 3:20 PM] ghaskins: What is the stack trace on the actual abort?",
                "ts": "1475274055.001875",
                "author_subname": "ghaskins",
                "channel_id": "C0J02NU1E",
                "channel_name": "general",
                "is_msg_unfurl": true,
                "text": "What is the stack trace on the actual abort?",
                "author_name": "Greg Haskins",
                "author_link": "https:\/\/hyperledgerproject.slack.com\/team\/ghaskins",
                "author_icon": "https:\/\/avatars.slack-edge.com\/2016-04-06\/32586535492_5cc4252021ffddc6d448_48.jpg",
                "mrkdwn_in": [
                    "text"
                ],
                "color": "D0D0D0",
                "from_url": "https:\/\/hyperledgerproject.slack.com\/archives\/general\/p1475274055001875",
                "is_share": true,
                "footer": "Posted in #general"
            }
        ],
        "ts": "1475340563.001890"
    },
    {
        "type": "message",
        "user": "U0KPFAZNF",
        "text": "<@U294C6FDW> is it hitting that error?",
        "ts": "1475341639.001891"
    },
    {
        "type": "message",
        "user": "U294C6FDW",
        "text": "yes it need to check if rowBytes is empty",
        "ts": "1475341742.001892"
    },
    {
        "type": "message",
        "user": "U0KPFAZNF",
        "text": "ok, this makes sense",
        "ts": "1475342279.001893"
    },
    {
        "user": "U2J8EAK99",
        "text": "<@U2J8EAK99|butch> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1475343793.001894"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "When trying to enroll a user I get...\n```ERROR: failed to register admin: Error: Identity or token does not match```\nbut I've checked the membersrvc.yaml file and the password is correct. Any ideas anyone?",
        "ts": "1475343920.001895"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "Quick overview of my setup:\nFabric VM running on a specific IP\n2. ssh then cd \/opt\/gopath\/src\/github.com\/hyperledger\/fabric\/build\/bin\n3. membersrvc &gt; membersrvc.log 2&gt;&amp;1&amp;\n4. peer node start --peer-chaincodedev\n\nHostOS\n1. node app calling the above on the assigned IP \n\nThe communication layer works as the error I'm getting is coming from the membersrvc (I think?)",
        "ts": "1475344163.001896"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "Basically I'm trying to get the chaincode_example02 running from a \"remote\" node app...",
        "ts": "1475344216.001897"
    },
    {
        "type": "message",
        "user": "U1CTBUNEN",
        "text": "<@U2CC1NDT2> I used to get this when attempting to enroll the same user twice",
        "ts": "1475344225.001898"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "aha - ok....",
        "ts": "1475344239.001899"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "so just for testing if I kill everything and delete the production dir and then start it all up and call the enroll once only from the node side it should work...?",
        "ts": "1475344299.001900"
    },
    {
        "type": "message",
        "user": "U1CTBUNEN",
        "text": "yeah",
        "ts": "1475344308.001901"
    },
    {
        "type": "message",
        "user": "U1CTBUNEN",
        "text": "<@U2CC1NDT2> the only 'advanced' samples I managed to run was the JS asset_management unit tests (via docker containers)",
        "ts": "1475344402.001902"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "ok - that worked although I got a `sending init` failed...",
        "ts": "1475344511.001903"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "but that's fine - the point was to check if the comms actually works - which I'm now satisfied that it does... thanks for pointer the one-time only enrollment requirement... forgot about that...",
        "ts": "1475344604.001904"
    },
    {
        "type": "message",
        "user": "U2CC1NDT2",
        "text": "what is strange though is I just ran the enrollment request again to check that it fails and it didn't... it went through fine",
        "ts": "1475344754.001905"
    },
    {
        "type": "message",
        "user": "U294C6FDW",
        "text": "I sent a fix last week when I encounter this issue.  they have not responded.",
        "ts": "1475347523.001906"
    },
    {
        "user": "U2JBNGHS9",
        "text": "<@U2JBNGHS9|syrohei> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1475378796.001907"
    },
    {
        "user": "U2JBS4S94",
        "text": "<@U2JBS4S94|pa7x> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1475389443.001908"
    }
]